### DISCUSSION TITLE
Добавить поле в таблицу кодом

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#343] [Как это сделать?] [answered]
Здравствуйте. Подскажите, пожалуйста, можно ли в таблицу циклов добавлять поле с нужним типом, настройками, кодом с помощью кода (кнопкой)?
спасибо

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Теоретически можно. На практике вы убъете базу. И надо понимать, что структура таблицы одинакова для циклов с одной версией.

Есть механизм версий, которые позволяют менять (добавлять/удалять поля и менять их состав) не изменяя предыдущие циклы, но создавать версию под конкретный цикл тоже плохая идея.

**Reply by tvideo** [reply]
ок. Понял. Спасибо
